include/chemistry/bcl_chemistry_fragment_stochastic_pose_optimizer.h

```diff
@@ -242,7 +242,7 @@ namespace bcl
       //! @param MOLECULE the molecule whose pose is to be perturbed
       //! @param CONF_ENSEMBLE optionally a conformer ensemble of the molecule to be perturbed
       //! @return returns the perturbed molecule
-      util::ShPtr< FragmentComplete> PerturbMoleculePose
+      util::ShPtr< FragmentComplete> PerturbMoleculePoseFiveDegreesMax
```

Comment:
Probably smart to rename this. It always caused issues with the formatter and usually I would manually change it after. I know the function is able to be used in the FragmentMapConformer class or one of those related drug design classes, but I am not sure if example functionality will change from this. I will check.